docs/source/conf.py

```diff
@@ -77,8 +77,15 @@
 # and we document the types anyway
 autodoc_typehints = "none"
 
+# Show docstring for special members
+autodoc_default_options = {
+    "special-members": True,
+    "exclude-members": "__init__",
+}
+
 # Fail on warnings & unresolved references etc
 nitpicky = True
+nitpick_ignore = [("py:class", "type")]
```

Comment:
! why? I removed it locally and didn't see anything fail related to this

Reply:
yea, I was just trying to fix that build warning..